### PR TITLE
change icon and its angle

### PR DESCRIPTION
**Pull request by palak199**
The earlier commits were not in accordance to rules of contribution so i am redoing it. Please review.
addresses issue #52

### CODE REVIEW

File: static/home/css/style.css [hunk 1]
```diff
@@ -748,6 +748,12 @@ nav{
 	z-index: 2;
 	/*cursor: pointer;*/
 }
+
+.con-sidenav i{
+	margin-right: 5px;
+	font-size: 25px;
```
Comment (garg3133) [inline static/home/css/style.css:754]: The icon size looks too big! I think 16px would be a reasonable font-size.

File: templates/home/index.html [hunk 2]
```diff
@@ -417,7 +417,7 @@ <h1 data-aos="fade-up">Contact Us</h1>
         </div>
     </section>
     <div class="con-sidenav">
-        <a href="#contact-us">Contact Us</a>
+        <a href="#contact-us"><i class="fa-rotate-90 fas fa-phone"></i>Contact Us</a>
```
Comment (garg3133) [inline templates/home/index.html:420]: Can you please put `fa-rotate-90` at the end of classes instead of at the start?

**Comment by garg3133** [comment]
@palak199 You are doing the same thing again!

**Comment by garg3133** [comment]
As now the font-size of the icon is reduced, can you please also fix the width of the label as there is a lot of empty space after `Contact Us`.

**Comment by palak199** [comment]
I have changed the width. Please check.

**Comment by garg3133** [comment]
@palak199 Can you please remove the commit a96a010 from this branch as this commit is unrelated to this issue.

**Comment by palak199** [comment]
> @palak199 Can you please remove the commit [a96a010](https://github.com/garg3133/JagratiWebApp/commit/a96a010221a8dc478892635583e39625b3dc7292) from this branch as this commit is unrelated to this issue.

I made another PR referencing this issue and that has only 2 files changed, neat and clean. Sorry for the inconvenience

**Pull request closed without merging**